packages/sui-widget-embedder/downloader/index.js

```diff
@@ -108,11 +108,22 @@
   }
 
   var pages = []
-  for (var key in pathnamesRegExp) {
-    if (window.location.pathname.match(new RegExp(pathnamesRegExp[key]))) {
-      pages.push(key)
+  for (var page in pageConfigs) {
+    var blacklistedRegExps = pageConfigs[page].blacklistedRegExps
+    var pathnameRegExp = pageConfigs[page].pathnameRegExp
+
+    if (
+      blacklistedRegExps &&
+      !blacklistedRegExps.some(regExp =>
```

Comment:
Arrow function might not be compatible with all browsers. Let's use a normal one.